bees/emailbee/emailbee.go

```diff
@@ -43,18 +44,32 @@ func (mod *EmailBee) Action(action bees.Action) []bees.Placeholder {
 
 	switch action.Name {
 	case "send":
-		to := ""
-		text := ""
-		subject := ""
+		var to, mailtext, subject string
 
 		action.Options.Bind("recipient", &to)
-		action.Options.Bind("text", &text)
+		action.Options.Bind("text", &mailtext)
 		action.Options.Bind("subject", &subject)
 
-		text = "Subject: " + subject + "\n\n" + text
-		auth := smtp.PlainAuth("", mod.username, mod.password, mod.server[:strings.Index(mod.server, ":")])
-		err := smtp.SendMail(mod.server, auth, mod.username, []string{to}, []byte(text))
-		if err != nil {
+		var host string
+		var port int
+		if strings.Index(mod.server, ":") != -1 {
```

Comment:
Instead of doing this manually, I'd suggest using net.SplitHostPort: https://golang.org/pkg/net/#SplitHostPort